### PR TITLE
Convert Google Doc Images to Gatsby Nodes

### PR DESCRIPTION
**Pull request by victsant**
- This will be enabled by **convertImgToNode** in the config

### CODE REVIEW

File: src/gatsby-node.js [hunk 1]
```diff
@@ -49,19 +61,48 @@ exports.sourceNodes = async (
...
+    let image = [];
+    for(document of googleDocsDocuments) {
+        const id = createNodeId(`GoogleDocs-${document.id}`);
+        let markdownNode = {
+          document,
+          id,
+          internal: {
+            type: "GoogleDocs",
+            mediaType: "text/markdown",
+        }
```
Comment (xuopled) [inline src/gatsby-node.js]: please use prettier to format the file

File: src/gatsby-node.js [hunk 2]
```diff
@@ -49,19 +61,48 @@ exports.sourceNodes = async (
...
-          type: "GoogleDocs",
-          mediaType: "text/markdown",
-          content: document.markdown,
-          contentDigest: createContentDigest(document.markdown),
-        },
-      })
-    })
+                    if (imageNode) {
+                       document.markdown = document.markdown.replace(url, imageNode.id);
+                    }else {
```
Comment (xuopled) [inline src/gatsby-node.js]: } else {

File: README.md [hunk 3]
```diff
@@ -195,6 +220,9 @@ JSON will be transformed to YAML and added to your markdown frontmatter and ovve
 
 > /!\ Do not use `id`, `name`, `description` or any `Google Docs` field you add to the config `fields` option
 
+### If convertImgToNode is enabled. You will need to search the id in the HTML file and replace it with Gatsby image tag
```
Comment (xuopled) [inline README.md]: Please add an example

**Comment by victsant** [comment]
@xuopled,

Updated the code and Readme file.

File: package.json [hunk 4]
```diff
@@ -1,12 +1,8 @@
 {
-  "name": "gatsby-source-google-docs",
-  "version": "1.7.0",
+  "name": "@victsant/gatsby-source-google-docs",
```
Comment (xuopled) [inline package.json]: Please don't update the package.json name and version.
Use yarn instead of NPN. I don't want package-lock.json

Comment (victsant) [reply package.json]: Sorry was testing something.

**Comment by xuopled** [comment]
Thank you for your work. I just release v1.9.0

**Comment by xuopled** [comment]
closes #13

**Pull request merged**